profiles/canopy_cloud_fabric.json

```diff
@@ -81,6 +81,42 @@
       "versions": [
 	  
       ]
+    },
+	{
+      "language": "binary",
```

Comment:
Please see https://github.com/stefan-kolb/paas-profiles#language for allowed languages.